include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -405,15 +411,53 @@ class LAPACKFullMatrix : public TransposeTable<number>
   void compute_lu_factorization ();
 
   /**
+   * Compute the Cholesky factorization of the matrix using LAPACK function Xpotrf.
+   *
+   * @note The factorization is stored in the lower diagonal.
+   */
+  void compute_cholesky_factorization ();
+
+  /**
+   * Estimate reciprocal of the condition number $1/k(A)$ in $L_1$ norm ($1/(||A||_1 ||A^{-1}||_1)$)
+   * of a SPD matrix using Cholesky factorization. This function can only
+   * be called if the matrix is already factorized.
+   *
+   * @note the condition number $k(A)$ can be used to estimate the numerical
+   * error related to the matrix inversion or the solution of the SLAE as
+   * <code>error = std::numeric_limits<Number>::epsilon * k</code>.
+   * Alternatively one can get the number of accurate digits
+   * <code>std::floor(std::log10(k))</code>.
+   *
+   * @param l1_norm Is the $L_1$ norm of the matrix before calling Cholesky
+   * factorization.
```

Comment:
how would one obtain this number? And can you mark it as `[in]` to make clear that it's something the user has to provide.